src/components/ebay-toast/index.marko

```diff
@@ -10,9 +11,9 @@ import processHtmlAttributes from "../../common/html-attributes"
     buttonPosition="right"
     on-modal-show("emit", "toast-show")
     on-modal-close("emit", "toast-close")
-    class=[
-        input.class,
-        "toast--transition"
-    ]>
+    class=[input.class, "toast--transition"]>
+    <@header class=[header.class, "toast__title"] ...header>
+        <${header.renderBody}/>
```

Comment:
This renderBody is not necessary, `renderBody` will be forwarded with `...header`.

`<@header class=[header.class, "toast__title"] ...header/>` should be fine.